ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,74 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    return visualMetrics.find((metric) => title.includes(metric));
+  };
+
+  anyAlertWithVideoResults = (alertSummary) => {
+    // For the moment the browsertime vismet are separate from the pageload ones
+    // that's why we need to filter them out. Also, we're retrieving the video results
+    // for regressions only
+    const any = alertSummary.alerts.filter(
+      (alert) =>
+        !this.containsVismet(alert.title) && alert.is_regression === true,
```

Comment:
I wonder 🤔... Couldn't we make use of the 'isRelevantAlert()' method here, as a filter predicate?